api/lib/src/models/property.dart

```diff
@@ -22,6 +23,7 @@ sealed class Property with _$Property {
   const factory Property.pen({
     @Default(5) double strokeWidth,
     @Default(0.4) double thinning,
+    @Default(0.5) double  straightLineTime,
```

Comment:
Properties are values shared between element and tool. I would remove it here and adding it to the tool itself

Reply:
if i remove it, receive an error

Reply:
Please remove it there, if you receive an error, you can ask on discord.